src/django_components/util/tag_parser.py

```diff
@@ -138,90 +172,109 @@ def take_while(tokens: Sequence[Union[str, Tuple[str, ...]]]) -> str:
 
         return result
 
-    # Parse
+    def parse_attr_parts() -> List[TagAttrPart]:
```

Comment:
This function was move from [line 189](https://github.com/EmilStenstrom/django-components/pull/857/files#diff-fcef21070412acf8475f5c6880736500491a9b4fbc5d77a37d46dd63814c158fL189) and  and updated.